### PR TITLE
Issue 4 now complete

### PR DESCRIPTION
**Pull request by jyaaan**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -39,6 +38,14 @@ app.get('/follow', (req, res) => {
   twitter.postFollow({}, error, success);
 })
 
+app.get('/ids', (req, res) => {
+  database.getUserIds('clients')
+    .then((result) => {
+      console.log(result[1].id);
+      res.send(result[1].id);
```
Comment (victormejia) [inline main.js:45]: only want one result?

Comment (jyaaan) [reply main.js:45]: The result array contains 1,500 ids so this was a way to test functionality without dumping a lot of text onto the screen.

Comment (victormejia) [reply main.js:45]: ah, gotcha 👍

**Review by victormejia — APPROVED** [review]
looks good

**Pull request merged**